ckanext/validation/tests/test_custom_checks.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from goodtables import validate
+from ckanext.validation.custom_checks import setup_custom_goodtables
+
+setup_custom_goodtables()
+
+
+# Validate
+def test_check_custom_constraint(log):
```

Comment:
These tests are copied and tweaked from upstream goodtables.  They should be run with pytest, explicitly specifying the test_custom_checks module.